samples/drivers/dac/README.rst

```diff
@@ -0,0 +1,32 @@
+.. _dac-sample:
+
+Digital-to-Analog Converter (DAC)
+#################################
+
+Overview
+********
+
+This sample demonstrates how to use the DAC driver API.
+
+Building and Running
+********************
+
+The DAC output is defined in the board's devicetree and pinmux file.
+
+The first supported board is the nucleo_l073rz. More boards will follow.
```

Comment:
Please reword to reflect current status.

Reply:
What do you mean exactly?

Reply:
"Only board nucleo_l073rz is supported for now"

Reply:
Ok, will update that with next force-push.